// This is the public API offered by your entity.
syntax = "proto3";

import "google/protobuf/empty.proto";
import "akkaserverless/annotations.proto";
import "google/api/annotations.proto";

package com.example;

message MyRequest {
    string name = 1;
}

message MyResponse {
    string text = 1;
}

service MyApi {
    rpc Hello(MyRequest) returns (MyResponse) {
        option (google.api.http) = {
            post: "/hello",
            body: "*"
        };  
    }
}